src/js/editor/todo_events.js

```diff
@@ -0,0 +1,54 @@
+import axios from '../axios-config';
+
+export function getAll() {
+  return new Promise((resolve, reject) => {
+    axios
+      .get('/lists')
+      .then(res => res.data)
+      .then(items => resolve(items))
+      .catch(err => console.log(err));
+  });
+}
```

Comment:
wszystkie metody `axios` zwracają Promisy. bez sensu jest to opakowywanie ich dodatkowe w `new Promsie`.
na dobrą sprawę to powinno wyglądać tak:
```js
export function getAll() {
  return axios.get('/lists')
}
```
a obsługa tego promisa:
`.then()` i `catch()` tam gdzie bezpośrednio wykonujesz ten kod